### PR TITLE
{CI} Live test optimize

### PR DESCRIPTION
**Pull request by wangzelin007**
**Related command**

**Description**

1. Support live test on multiple platforms ( MacOS / Windows / Ubuntu )
2. Support extension live test
3. Send live test data to kusto database.
4. Add custom cleanup scripts to clean up residual resources
5. Change `az login -u -p` to `az ligin with service principle`.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

**Review by kairu-ms — APPROVED** [review]

File: scripts/live_test/generate_index.py [hunk 1]
```diff
@@ -148,17 +163,19 @@ def render(data, container, container_url, testdata, USER_REPO, USER_BRANCH, COM
...
             if name.startswith(module + '.'):
                 display_name = 'report'
-                if 'parallel' in name:
-                    display_name = 'parallel'
-                elif 'sequential' in name:
-                    display_name = 'sequential'
+                # if 'parallel' in name:
+                #     display_name = 'parallel'
+                # elif 'sequential' in name:
+                #     display_name = 'sequential'
```
Comment (bebound) [inline scripts/live_test/generate_index.py:178]: Any reason to left this comment here?

Comment (wangzelin007) [reply scripts/live_test/generate_index.py:178]: In the past, each module will generate two reports, one is parallel and the other is sequential. If it is multi-platform, there will be more reports. Now I have integrated them into one report, so these codes are not needed.
Beofre:
image
After:
image

**Review by bebound — APPROVED** [review]

**Pull request merged**